project-775f91f9-2740-42c4-9764-2f12bd68318f/.monokle

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I'm not sure what this file is? Maybe you added it by mistake

Reply:
it was added by mistake, I removed the file